buildscripts/github/setup_platform.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -x
+
+PLATFORM=$1
+
+echo "Setting up platform-specific requirements for ${PLATFORM}"
+
+case "${PLATFORM}" in
+    "osx-64")
+        echo "Setting up macOS SDK for osx-64 build"
+        sdk_dir="buildscripts/github"
+        mkdir -p "${sdk_dir}"
+
+        # Download SDK
+        echo "Downloading MacOSX10.10.sdk.tar.xz"
+        wget -q https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX10.10.sdk.tar.xz
+
+        # Verify checksum
+        echo "Verifying SDK checksum"
+        shasum -c "${sdk_dir}/MacOSX10.10.sdk.checksum"
```

Comment:
```suggestion
        shasum -c "${sdk_dir}/MacOSX10.10.sdk.checksum" || exit 1
```

Besides all the OOB discussion regarding using `set -e` and inheriting the flags from the parent shell, let's use the above construct in order to make explicit (for the reader) that this command absolutely must terminate execution of the script

Reply:
sure, will update